cves/CVE-2016-3092.yml

```diff
@@ -122,7 +128,7 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
+  answer: 
   date:
   automated:
   contest:
```

Comment:
Do you have any information on its discovery? I believe some of these fields need to be filled in to pass the build as well.